### PR TITLE
Block Nav

### PR DESCRIPTION
**Pull request by maxwensemble**
Test URLs:
- https://block-nav--brother-usa-franklin-demo--ensemblesystems.hlx.page/

Finished:
- header & nav bar

Working on:
- Slick slider

Changes:
- Structure 
  - header & nav bar block
  - nav bar menu and sub menu content
- style header & nav bar block
- Add temp code for slick slider

### CODE REVIEW

File: blocks/brand/brand.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+                topLevelBrandMenuItems.forEach(topLevelBrandMenuItem => {
+                    if (item !== topLevelBrandMenuItem && !topLevelBrandMenuItem.contains(item)) {
+                        topLevelBrandMenuItem.classList.remove('clicked');
+                    }
+                });
+
+                const parentLi = event.currentTarget.parentNode;
+                const siblingUl = Array.from(parentLi.children).find(child => child.tagName === 'UL');
+                console.log('siblingUl:', siblingUl);
```
Comment (alexm-ens) [inline blocks/brand/brand.js:64]: dont forget to remove console.logs before final draft

File: blocks/home-and-business/home-and-business.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    block.innerHTML = '';
+    
+    const homeLink = document.createElement('a');
+    homeLink.classList.add('home-link');
+    homeLink.href = `${window.location.origin}/`;
+    homeLink.textContent = 'For Home';
+
+    const businessLink = document.createElement('a');
+    businessLink.classList.add('business-link');
+    businessLink.href = `${window.location.origin}/`;
```
Comment (alexm-ens) [inline blocks/home-and-business/home-and-business.js:15]: perfect!

File: scripts/ffetch.js
Comment (alexm-ens) [inline scripts/ffetch.js:1]: we wont need this file i think. It was probly for excel spreadsheet?

File: blocks/brand/brand.css [hunk 3]
```diff
@@ -0,0 +1,161 @@
+#brand-menu {
+    display: none;
```
Comment (alexm-ens) [inline blocks/brand/brand.css:2]: is that still needed?

File: blocks/brand/brand.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                    if (item !== topLevelBrandMenuItem && !topLevelBrandMenuItem.contains(item)) {
+                        topLevelBrandMenuItem.classList.remove('clicked');
+                    }
+                });
+
+                const parentLi = event.currentTarget.parentNode;
+                const siblingUl = Array.from(parentLi.children).find(child => child.tagName === 'UL');
+                if (siblingUl) {
+                    console.log('The clicked item has a <ul> sibling.');
+                    parentLi.classList.toggle('clicked');
```
Comment (alexm-ens) [inline blocks/brand/brand.js:66]: console log can be removed

File: blocks/brand/brand.css (hunk 3 above)
Comment (alexm-ens) [reply blocks/brand/brand.css:2]: ok i checked the doc and see why this is here. I checked js file and im wondering if u delete this h6 from the doc completely will this break anything? From where i stand i feel like there should be no issues. 

But yeah if im wrong and it breaks the layout keep it there, no problem

File: blocks/brand/brand.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+                } else {
```
Comment (alexm-ens) [inline blocks/brand/brand.js:68]: probably dont need this else

File: blocks/home-and-business/home-and-business.css [hunk 6]
```diff
@@ -0,0 +1,68 @@
+.home-and-business-container {
+    /*hacky css to get things working. remove when done*/
+    padding-top: 0%;
```
Comment (alexm-ens) [inline blocks/home-and-business/home-and-business.css:3]: still needed?

**Review by alexm-ens — CHANGES_REQUESTED** [review]
requesting changes for removing console.log and else statement. 

For css comments, just check if can be removed if not no problem

File: blocks/home-and-business/home-and-business.css (hunk 6 above)
Comment (maxwensemble) [reply blocks/home-and-business/home-and-business.css]: not needed

File: blocks/brand/brand.css (hunk 3 above)
Comment (maxwensemble) [reply blocks/brand/brand.css:2]: This one is needed because it will break the layout

File: blocks/home-and-business/home-and-business.css [hunk 7]
```diff
@@ -0,0 +1,62 @@
+main>.section {
+    /*hacky css remove when done*/
```
Comment (alexm-ens) [inline blocks/home-and-business/home-and-business.css:2]: ok that probably will need to go when working on index page

**Review by alexm-ens — APPROVED** [review]
well done!

**Pull request merged**